src/formatDate.js

```diff
@@ -8,7 +8,54 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const [oldData, idFromYear, oldYear, fromSymbol] = getFormat(fromFormat);
+  const [newData, , newYear, toSymbol] = getFormat(toFormat);
+
+  const fromDate = date.split(fromSymbol);
```

Comment:
The `split` method assumes that the date string will always contain the `fromSymbol`. If the input date does not match the expected format, this could lead to unexpected results or errors. Consider adding validation or error handling for the input date.